Assets/LeapMotionModules/ElementRenderer/Scripts/Features/Mesh/ProceduralMeshes/ProceduralPanel.cs

```diff
@@ -4,35 +4,80 @@
 using Leap.Unity.Attributes;
 
 public class ProceduralPanel : ProceduralMeshSource {
```

Comment:
In keeping with the naming convention, it might be good to add "LeapGui" or "LeapElement" as a prefix to the procedural panel; keeps it from being confused with other non-Leap Element Renderer Procedural Panel/Mesh Generators